vue-project-stub-no-pwa/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  preset: '@vue/cli-plugin-unit-jest/presets/typescript-and-babel'
+}
```

Comment:
Should this have `transformIgnorePatterns: []` as well?

Reply:
Yes, it should indeed! Will update